website/content/en/status/report-2023-04-2023-06/mfsbsd.adoc

```diff
@@ -1,28 +1,28 @@
-=== Integrate mfsBSD into the release building tools 
+=== Integrate mfsBSD into the release building tools
 
-Links: + 
-link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[Wiki Article] URL: link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[] 
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[Wiki Article] URL: link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[]
 
-Contact: Soobin Rho <soobinrho@FreeBSD.org> 
+Contact: Soobin Rho <soobinrho@FreeBSD.org>
```

Comment:
Whilst this email address `@freebsd.org` is mentioned in the wiki page above, it is not found in the places (via <https://wiki.freebsd.org/Developers#DeveloperLists>) that are normally associated with contributors to FreeBSD. 

Please, can someone confirm that the email address is valid?

Reply:
@Jehops @jurajlutter ▲

Reply:
@soobinrho

Reply:
It's valid.  All GSoC students have the option to create @FreeBSD.org aliases.

Reply:
@Jehops @jurajlutter : I believe GSoC students should be added at least to https://docs.freebsd.org/en/articles/contributors/ , probably in the additional contributors list.